lebedigital/openbis/README.md

```diff
@@ -0,0 +1,140 @@
+  - _Example_: Concrete_Compression_Experiment_17_10_2022
+- **Dataset** - An openBIS abstraction containing your data files like Excel sheets or csvs.
+  - _Example_: Concrete_Compression_Experiment_17_10_2022_RAW_DATA
+- **Sample / Object Type** - A blueprint you need to define in order to upload Samples/Objects of real world experiments.
+  - _Example_: EXPERIMENTAL_STEP_CONCRETE
+- **Property** - One piece of metadata you use do describe your Samples/Objects. 
+  - _Example_: Name of your sample
+- **Parents** of a Sample - You can define another Sample to be a parent of a different sample.
+  - _Example_: A concrete mixture could be a **parent** of a concrete compression experiment
+
+> You can look at the definition from pybis [here](https://pypi.org/project/PyBIS/)
+
+## Structure
+
+### Interbis
+
+Is an extension of the existing Openbis class defined in pybis and as such extends all pybis functionality.
+Interbis provides additional functionality for getting an overview of the datastore and find out what the structure of the Database looks like.
+Interbis also makes it easier to search for existing samples and retrieve their Identifiers from their metadata like Name.
+
+### ExpStep
+
+Is a Class which provides functionality to define Samples as they are accepted by pybis and in turn the openBIS datastore.
+Example uses of the Class include uploading and retrieving samples and datasets from the Database.
+I recommend deriving the class in your personal workflow and tweaking it to make it easier to write data processing and uploading scripts. 
+
+
+## Example Workflow of Interbis/ExpStep - Concrete Measuring Project
+
+### Start with connecting yourself to the Database
+
+```python
+from lebedigital.openbis.interbis import Interbis
+
+o = Interbis('https://yourdatabase.com')
+o.connect_to_datastore()
+```
+
+### Then you can define a blueprint from which you will create the Experimental Steps you have done in your research - create a new **Sample Type**
+
+```python
+# In order to define a new Sample Type use the following method
+
+o.create_sample_type(
+    sample_code='EXPERIMENTAL_STEP_CONCRETE_TEST',
+    sample_prefix='CONCRETE_TEST',
+    sample_properties={
+        'property1': ['VARCHAR', 'my_label1', 'my_description1'],
+        'property2': ['INTEGER', 'my_label2', 'my_description2'],
+        'property3': ['FLOAT', 'my_label2', 'my_description2']
+    }
+)
+```
+> You can find all the possible property types like 'VARCHAR' or 'FLOAT' [here](https://pypi.org/project/PyBIS/) under 'create property types'
+
+> You can also define the Sample Type in the Web-View of the Datastore
+### Now you can create the Experimental Step describing your research experiment. Start by declaring an object of the ExpStep class, which you want to upload to the Datastore
```

Comment:
change heading as above and add sentence in text body

add: For each measured experiment an object/sample of the corresponding type (EXPERIMENTAL_STEP_XY) has to be created. Therefore, create for each one instances of the ExpStep class which then can be uploaded into the Datastore.

Reply:
Changed and added